python/ray/worker.py

```diff
@@ -1806,7 +1806,8 @@ def process_task(task):
       function_name, _ = (worker.functions[task.driver_id().id()]
                           [function_id.id()])
       contents = {"function_name": function_name,
-                  "task_id": task.task_id().hex()}
+                  "task_id": task.task_id().hex(),
+                  "worker_id": worker.worker_id.hex()}
```

Comment:
Looks like this is failing. You probably just need to get rid of the `.hex()` since I think `worker_id` is already a string.